app/backend/user/serializers.py

```
@@ -142,8 +172,42 @@ class Meta:
             "dru",
             "is_verified",
             "is_admin",
+            "profile_image",
+            "id_card_image",
         ]
 
+    def validate_uploaded_image_file(self, image_file):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)